### PR TITLE
ENH: DelayBase offset

### LINKED ISSUE #568: Allow offsets/zeroing in delay stage class
## Expected Behavior

Have a user-configurable offset in the delay stage class

## Current Behavior

We just multiply the real motor number to get the time number

### PR DESCRIPTION
**Pull request by klauer**
## Description
* Add `set_current_position` and `user_offset` to `DelayBase`

## Motivation and Context
Closes #568 

## How Has This Been Tested?
* Basic unit tests
* Interactively

## Where Has This Been Documented?
* Docstrings
* Release notes

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]
This looks clean to me

File: pcdsdevices/pseudopos.py [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
 import numpy as np
+import ophyd
 from ophyd.device import Component as Cpt
 from ophyd.device import FormattedComponent as FCpt
 from ophyd.pseudopos import (PseudoPositioner, PseudoSingle,
                              pseudo_position_argument, real_position_argument)
 from scipy.constants import speed_of_light
 
 from .interface import FltMvInterface
+from .signal import Signal
```
Comment (ZLLentz) [inline pcdsdevices/pseudopos.py:13]: Any reason to pull this from local instead of from ophyd?

Comment (klauer) [reply pcdsdevices/pseudopos.py]: Haven't a clue why I did that; pushed a fix.

Comment (ZLLentz) [reply pcdsdevices/pseudopos.py]: :+1:

**Comment by ZLLentz** [comment]
LGTM, feel free to merge after Travis

**Comment by ZryletTC** [comment]
Could the offset behavior in DelayBase be changed so that it edits the offset of the underlying motor instead of having two layers of offset? It gets a little confusing/messy.

**Pull request merged**